pennylane/labs/resource_estimation/__init__.py

```diff
@@ -40,8 +40,14 @@
 from .ops import (
     ResourceCNOT,
     ResourceControlledPhaseShift,
+    ResourceGlobalPhase,
     ResourceHadamard,
+    ResourceIdentity,
+    ResourceRot,
+    ResourceRX,
     ResourceRZ,
+    ResourceRY,
+    ResourceS,
```

Comment:
Don't forget to add these to the doc-string above so that they render in the docs